xla/service/gpu/cublas_cudnn.cc

```diff
@@ -75,6 +75,8 @@ const absl::string_view kCudnnConvReorderFilterAndBiasCallTarget =
 const absl::string_view kCudnnNormCallTarget = "__cudnn$norm";
 
 // fMHA forward call targets.
+const absl::string_view kCudnnfMHASoftmaxF8CallTarget =
+    "__cudnn$fmhaSoftmax$f8";
```

Comment:
Should this be __cudnn$fmhaSoftmaxF8?

Reply:
I follow the convention in "__cublas$lt$matmul$f8"; so there is a dollar sign.